src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -2845,7 +2845,7 @@ because xbuild doesn't support framework reference assemblies.
 	<ItemGroup>
 		<AndroidNativeLibrary Include="$(MSBuildThisFileDirectory)\lib\*\libsqlite3_xamarin.so" Condition="$(_Assemblies.Contains('Mono.Data.Sqlite.dll'))" />
 		<AndroidNativeLibrary Include="$(MSBuildThisFileDirectory)\lib\*\libMonoPosixHelper.so" Condition="$(_Assemblies.Contains('Mono.Posix.dll'))" />
-		<AndroidNativeLibrary Include="$(MSBuildThisFileDirectory)\lib\*\libmono-native.so" Condition="$(_Assemblies.Contains('mscorlib.dll'))" />
+		<AndroidNativeLibrary Include="$(MSBuildThisFileDirectory)\lib\*\libmono-native.so"  />
```

Comment:
I believe that this line is superfluous with `BuildApk.cs` line 415: Only one of the two lines should be needed, not both, as `@(AndroidNativeLibrary)` should *cause* an `AddNativeLibrary()` invocation to occur.